papers/bradley_dice/55_bradley_dice.rst

```diff
@@ -0,0 +1,498 @@
+:author: Bradley D. Dice
+:email: bdice@umich.edu
+:institution: Department of Physics, University of Michigan, Ann Arbor
+
+:author: Brandon L. Butler
+:email: butlerbr@umich.edu
+:institution: Department of Chemical Engineering, University of Michigan, Ann Arbor
+
+:author: Vyas Ramasubramani
+:email: vramasub@umich.edu
+:institution: Department of Chemical Engineering, University of Michigan, Ann Arbor
+
+:author: Alyssa Travitz
+:email: atravitz@umich.edu
+:institution: TODO Department, University of Michigan, Ann Arbor
+
+:author: Michael M. Henry
+:email: mikehenry@u.boisestate.edu
+:institution: Micron School of Materials Science and Engineering, Boise State University
+
+:author: Hardik Ojha
+:email: hojha@ee.iitr.ac.in
+:institution: TODO Department, Indian Institute of Technology Roorkee
```

Comment:
@kidrahahjo Can you update this with how you'd like your institution/department to be listed?